.github/workflows/e2e.yml

```diff
@@ -7,7 +7,6 @@ jobs:
 
     env:
       DATABASE_URL: postgresql://postgres:@localhost:5432/calendso
-      NODE_ENV: test
```

Comment:
This was the magic random line that made it all crash and burn (note that having `NODE_ENV=test` locally and running the exact same command actually worked 🥲🔫)

Reply:
🤭